webpack.config.js

```diff
@@ -0,0 +1,81 @@
+const webpack = require('webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = env => {
+	env = env || {};
+	const config = {
+    mode: env.mode || 'production',
+		entry: './src/index.js',
+		output: {
+			path: path.resolve(__dirname, 'dist'),
+			filename: 'bundle.js'
+    },
+		module: {
+			rules: [
+				{
+          test: /\.html$/,
+					use: [
+						{ loader: 'babel-loader' },
+						{
+              options: {
+                processStyleLinks: true
+              },
+              loader: 'polymer-webpack-loader'
+            }
+					]
+        },
+				{
+          test: /polymer\.html$/,
+          include: [
+            path.resolve(__dirname, './node_modules/@banno/polymer')
+          ]
+        },
+				{
+          test: /\.js$/,
+          use: 'babel-loader',
+        },
+        {
+          test: /\.css$/,
+          use: 'css-loader'
+        },
+        {
+          test: /\.(png|jpg|gif)$/,
+          use: 'file-loader'
+        }
+			]
+		},
+		devServer: {
+			contentBase: path.join(__dirname, 'dist'),
+			compress: true,
+			port: 1820
+
+    },
+    devtool: 'source-map',
+		plugins: [
+      new webpack.NormalModuleReplacementPlugin(
```

Comment:
This doesn't need changed, I just wanted to point out here that this is why the icons importing polymer.html instead of polymer-element.html isn't a problem. This is changing that path so it actually is requiring polymer-element.html